.github/workflows/docker_snapshot.yml

```diff
@@ -0,0 +1,81 @@
+name: Build internal snapshot image
+
+on:
+  push:
+    branches:
+      - 'master'
+      - '[0-9]+.[0-9]+.z'
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'hazelcast'
+    steps:
+      - name: Checkout hazelcast-snapshot repo
+        uses: actions/checkout@v2
+        with:
+          repository: hazelcast-dockerfiles/hazelcast-snapshot
+          ref: master
+          path: hazelcast-snapshot
+      - name: Checkout hazelcast-enterprise-snapshot repo
+        uses: actions/checkout@v2
+        with:
+          repository: hazelcast-dockerfiles/hazelcast-enterprise-snapshot
+          ref: master
+          path: hazelcast-enterprise-snapshot
+      - name: Checkout hazelcast repo
+        uses: actions/checkout@v2
+        with:
+          repository: hazelcast/hazelcast
+          path: hazelcast
+      - name: Checkout hazelcast-enterprise repo
+        uses: actions/checkout@v2
+        with:
+          repository: hazelcast/hazelcast-enterprise
+          ref: ${{ github.ref }}
+          path: hazelcast-enterprise
+          token: ${{ secrets.GH_PAT }}
+
+      - name: Set up Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```

Comment:
Just to understand better, why do we need QEMU?

Reply:
Good question - I'll remove the action.

I added it "just in case"  - to be prepared to build snapshots for additional platforms. But then we would still need to add buildx parameters so the QEMU setup here is useless.
https://github.com/marketplace/actions/build-and-push-docker-images#usage